### PR TITLE
OpenNLP 1.9.1 Upgrade

### PR DESCRIPTION
**Pull request by NightOwl888**
This upgrades the project to OpenNLP 1.9.1. Please update the package on NuGet.

Also, do note that the "official" IKVM project is dead, but there are some updates to it happening at: https://github.com/wwrd/ikvm8. In particular, there has been some progress on getting IKVM to work with .NET Standard: https://github.com/wwrd/ikvm8/issues/3. You might consider targeting [IKVM.WINDWARD](https://www.nuget.org/packages/IKVM.WINDWARD/) instead of IKVM so when these changes are complete this project will support .NET Standard (we will need this for OpenNLP).

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -1,6 +1,11 @@
 @echo off
 cls
 
+.paket\paket.bootstrapper.exe
+if errorlevel 1 (
+  exit /b %errorlevel%
+)
+
```
Comment (sergey-tihon) [inline build.cmd]: There is no need to  commit `paket.bootstrapper.exe`.
Note that `paket.exe` that you removed has very similar file size
OpenNLP_1_9_1_Upgrade_by_NightOwl888_·_Pull_Request__4_·_sergey-tihon_OpenNLP_NET

So `paket.exe` is not real Paket, it was bootstrapper used in magic mode
https://fsprojects.github.io/Paket/bootstrapper.html#Magic-mode

Comment (NightOwl888) [reply build.cmd]: No problem. The commit can easily be reverted/excluded.

**Comment by sergey-tihon** [comment]
Thank you @NightOwl888 
New version released on NuGet - https://www.nuget.org/packages/OpenNLP.NET/1.9.1

**Comment by NightOwl888** [comment]
Wow, now that was fast! Thanks.

**Pull request merged**